fc/templates/unused/TestClampingToInitialValue.pyx

```diff
@@ -0,0 +1,135 @@
+# cython: profile=True
```

Comment:
I suspect these were useful when you were first figuring out the template, but aren't really needed now. However, let's keep them in this PR and delete when we're a bit further forward with all the code generation features.

Reply:
Agreed! https://github.com/ModellingWebLab/weblab-fc/issues/88